### DISCUSSION TITLE
Where is the video stored?

### DISCUSSION THREAD

**Discussion by markabels** [aclap-dev/video-downloadhelper#935] [Help and Q&A]
### Your question

I downloaded a video successfully and now cannot find it.  Where does VDH store videos by default (it is not in my downloads file)?

### Addon & CoApp version and details

Version 9.0.2.12 (stable), target mozilla, locale en-US.  Companion App installed (v2.0.19)

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You can find an extensive, relevant discussion over here: #744.  In addition, after a download completes, VDH leaves a green entry in its menu:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/f0346618-89bd-4e8c-b94c-3216d79f7ed0)

If you click the control indicated in that image, VDH will open a new File Explorer (file manager) window with that directory displayed.  Ummm...... Usually.  I have a File Manager window open pretty much all the time with multiple directories in the tabs.  One of those directories happens to be one into which I downloaded something.  But clicking that VDH control just switched focus to the File Explorer window I already had open.  It did not also tab over to my download directory.  I would say this is a shortcoming in the API Windows offers for the File Manager.  There are several shortcomings of the Windows 11 File Manager that I have encountered.  This appears to be one of them.  So if this VDH control takes you to some mysterious directory, tab through all your open tabs within the File Manager window.  One of them might be your directory.

Failing that, the setting in VDH for `Download directory` might be your target directory if you last did a `Download` function.  If you last did a `Download As` function, that won't be true.  But if you start to launch another `Download As`, the file selection dialog will open on whatever directory you most recently downloaded into.  You don't need to actually do that download.  You can cancel the file selection dialog.  But it would have shown you where your most recent download went to.  There is a lot more background information on this in the discussion I cited above.

**Reply by markabels** [reply]
Thanks for your help and the very interesting discussion.  I have been a VDH user for years, but I wasn't around for all that early history.

Sometimes I amaze myself with my own thick-skulledness. I had to reinstall VDH and I had no trouble downloading and installing Co-App (I knew what that is and why it was needed), no trouble finding the settings gear and no trouble opening "More Settings."  But despite it being it right in front of my face, I didn't notice that you could scroll down!

Anyway, problem now solved, I've learned new skills, I discovered the file location where the videos have been stored for years (I picked that location so many years ago that I had forgotten it), and I am back at 100% functionality.  A good day, with your help.
